src/Uno.UI/KeyboardListener.Android.cs

```diff
@@ -0,0 +1,102 @@
+﻿using Windows.UI.Xaml;
+using Windows.UI.Xaml.Input;
+using Android.Views;
+using System;
+using Uno.UI.Extensions;
+
+namespace Uno.UI
+{
+	internal class KeyboardListener
+	{
+		private UIElement _rootContent;
+		private UIElement _focusedElement;
+
+		public void SetRootContent(UIElement rootContent)
+		{
+			if (_rootContent == rootContent)
+			{
+				return; // nothing to do
+			}
+
+			if (_rootContent != null)
+			{
+				rootContent.GotFocus -= OnFocus;
```

Comment:
I'm guessing these should be `_rootContent`?

Reply:
Fixed.